config/dev.exs

```diff
@@ -1,7 +1,7 @@
 use Mix.Config
 
 config :kafka_ex,
-  brokers: [{"ip-10-0-0-49.ec2.internal", 9092}, {"ip-10-0-0-248.ec2.internal", 9092}],
+  brokers: [{"ip-10-0-0-166.ec2.internal", 9092}],
```

Comment:
I actually tried it last Friday and it worked fine, I'll try it again and after that will switch to it.